src/main/external-resources/renderers/Roku-TV8.conf

```diff
@@ -26,25 +24,18 @@ HalveBitrate = true
 MediaInfo = true
 
 # Supported video formats:
-Supported = f:3gp|3g2         v:divx|h264|mjpeg|mp4|mpeg1|mpeg2|vp6|wmv   a:aac|aac-he|ac3|eac3|lpcm|mp3|wma   m:video/3gpp
-Supported = f:avi             v:divx|h264|mjpeg|mp4|mpeg1|mpeg2|vp6|wmv   a:aac|aac-he|ac3|eac3|lpcm|mp3|wma   m:video/avi
-Supported = f:flv             v:divx|h264|mjpeg|mp4|mpeg1|mpeg2|vp6|wmv   a:aac|aac-he|ac3|eac3|lpcm|mp3|wma   m:video/x-flv
-Supported = f:mkv             v:divx|h264|mjpeg|mp4|mpeg1|mpeg2|vp6|wmv   a:aac|aac-he|ac3|eac3|lpcm|mp3|wma   m:video/x-matroska
-Supported = f:mov             v:divx|h264|mjpeg|mp4|mpeg1|mpeg2|vp6|wmv   a:aac|aac-he|ac3|eac3|lpcm|mp3|wma   m:video/quicktime
-Supported = f:mp4             v:divx|h264|mjpeg|mp4|mpeg1|mpeg2|vp6|wmv   a:aac|aac-he|ac3|eac3|lpcm|mp3|wma   m:video/mp4
-
-# Supported = f:mpegps|mpegts   v:divx|h264|mjpeg|mp4|mpeg1|mpeg2|vp6|wmv   a:aac|aac-he|ac3|eac3|lpcm|mp3|wma   m:video/mpeg
-Supported = f:webm            v:vp8                                       a:vorbis                             m:video/webm
-Supported = f:wmv             v:divx|h264|mjpeg|mp4|mpeg1|mpeg2|vp6|wmv   a:aac|aac-he|ac3|eac3|lpcm|mp3|wma   m:video/x-ms-wmv
-
-Supported = f:mpegps     v:divx|h264|mjpeg|mp4|mpeg1|mpeg2|sor|vc1|vp6|wmv        a:aac|aac-he|ac3|adpcm|dts|eac3|lpcm|mp3|wma   m:video/mpeg
-Supported = f:mpegts     v:divx|h264|h265|mjpeg|mp4|mpeg1|mpeg2|sor|vc1|vp6|wmv   a:aac|aac-he|ac3|adpcm|dts|eac3|lpcm|mp3|wma   m:video/mpeg
-
-MPEGTS-H265-AC3    H.265 (HEVC)    AC-3    MPEG-TS     Only FFmpeg outputs H.265, others will output H.264
-MPEGTS-H265-AAC    H.265 (HEVC)    AAC     MPEG-TS     Only FFmpeg outputs H.265, others will output H.264
+Supported = f:avi             v:divx|h264|mjpeg|mp4|mpeg1|mpeg2|vp6|wmv   a:aac-lc|he-aac|ac3|eac3|lpcm|mp3|wma   m:video/avi
+Supported = f:flv             v:divx|h264|mjpeg|mp4|mpeg1|mpeg2|vp6|wmv   a:aac-lc|he-aac|ac3|eac3|lpcm|mp3|wma   m:video/x-flv
+Supported = f:mkv             v:divx|h264|mjpeg|mp4|mpeg1|mpeg2|vp6|wmv   a:aac-lc|he-aac|ac3|eac3|lpcm|mp3|wma   m:video/x-matroska
+Supported = f:mov             v:divx|h264|mjpeg|mp4|mpeg1|mpeg2|vp6|wmv   a:aac-lc|he-aac|ac3|eac3|lpcm|mp3|wma   m:video/quicktime
+Supported = f:mp4             v:divx|h264|mjpeg|mp4|mpeg1|mpeg2|vp6|wmv   a:aac-lc|he-aac|ac3|eac3|lpcm|mp3|wma   m:video/mp4
+Supported = f:mpegts          v:h264                                      a:aac-lc-lc|lpcm|mp2|mp3        n:2     m:video/mpeg
+Supported = f:webm            v:vp8                                       a:vorbis                                m:video/webm
+Supported = f:wmv             v:divx|h264|mjpeg|mp4|mpeg1|mpeg2|vp6|wmv   a:aac-lc|he-aac|ac3|eac3|lpcm|mp3|wma   m:video/x-ms-wmv
+Supported = f:3gp|3g2         v:divx|h264|mjpeg|mp4|mpeg1|mpeg2|vp6|wmv   a:aac-lc|aac-lc|ac3|eac3|lpcm|mp3|wma   m:video/3gpp
 
 # Supported audio formats:
-Supported = f:aac    m:audio/x-m4a
+Supported = f:aac-lc m:audio/x-m4a
```

Comment:
Here the "aac" was correct because this is a description of the file extension. @Nadahar was refering only to the audio codecs `a:aac`